lib/src/regex/get_utils_extensions.dart

```diff
@@ -32,6 +32,7 @@ extension GetStringUtils on String {
   bool get isPassport => GetUtils.isPassport(this);
   bool get isCurrency => GetUtils.isCurrency(this);
   bool isCpf(String s) => GetUtils.isCpf(this);
+  bool isCnpj(String s) => GetUtils.isCpf(this);
```

Comment:
bool isCnpj(String s) => GetUtils.isCnpj(this); 

please, fix the second parameter